### PR TITLE
CBV support - we have a camera now

### PR DESCRIPTION
**Pull request by ttanatb**
## Feature/Issue
We didn't have a camera

## Implementation/Solution
We have a camera now (I did a lot of heap stuff)

### CODE REVIEW

File: Light Vox Engine/ObjLoader.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <DirectXMath.h>
+#include <vector>
```
Comment (BenjaFriend) [inline Light Vox Engine/ObjLoader.h:3]: Shouldn't object loader be in the `Utils` folder ?

File: Light Vox Engine/Engine.cpp [hunk 2]
```diff
@@ -117,7 +121,28 @@ HRESULT Engine::Run()
 		}
 		else
 		{
-			graphics->Update();
+            //DEBUG CODE for basic transform update;
+            DirectX::XMFLOAT4X4 transforms[1];
+            static float r = 0;
+            DirectX::XMFLOAT3 vecUp = DirectX::XMFLOAT3(0, 1, 0);
```
Comment (arb7887) [inline Light Vox Engine/Engine.cpp]: Multiple up vectors in this scope, maybe avoidable?

File: Light Vox Engine/ObjLoader.h (hunk 1 above)
Comment (ttanatb) [reply Light Vox Engine/ObjLoader.h:3]: It only really one application in graphics and we won't have it for long anyways so I think it's fine where it is

**Comment by ttanatb** [comment]
Ready for re-review

**Review by BenjaFriend — APPROVED** [review]
Changes look good! ⛵️

**Pull request merged**